### PR TITLE
Pdf compliance

### PR DESCRIPTION
**Pull request by voisardf**
Adaptions of the PDF to the latest federal specifications. more to come

### CODE REVIEW

File: crdppf/lib/content.py [hunk 1]
```diff
@@ -561,11 +564,11 @@ def get_content(id, request):
         "outputFormat": "pdf"
     }
 
-    # import json
+    import json
```
Comment (voisardf) [inline crdppf/lib/content.py]: will be commented again before merging

File: print/print-apps/crdppf/config.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-throwErrorOnExtraParameters: true
+throwErrorOnExtraParameters: false
```
Comment (kalbermattenm) [inline print/print-apps/crdppf/config.yaml]: Is that OK for you ? (or have you changed that for testing purposes?)

Comment (voisardf) [reply print/print-apps/crdppf/config.yaml]: Yes, I prefer it to be set to false by default and activate it, in case of errors 👍

**Comment by voisardf** [comment]
thanks, merging

**Pull request merged**